kolibri/plugins/learn/assets/src/views/recommended-subpage/index.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+
+  <content-card-grid
+    :contents="recommendations"
+    :gen-content-link="genContentLink"
+    :header="header"
+    :subheader="$tr('pageSubheader', {numOfItems: recommendations.length})"/>
+    <!-- TODO make header dynamic -->
+
+</template>
+
+
+<script>
+
+  const { PageNames } = require('../../constants');
+
+  module.exports = {
+    $trNameSpace: 'recommendedSubpage',
+    $trs: {
+      popularPageHeader: 'Popular',
+      resumePageHeader: 'Resume',
+      nextStepsPageHeader: 'Next steps',
+      overviewPageHeader: 'Overview',
+      pageSubheader: '{numOfItems, number} items',
+    },
+    components: {
+      'content-card-grid': require('../content-card-grid'),
+    },
+    computed: {
+      header() {
+        // IDEA encode this information in the route?
+        switch (this.pageName) {
```

Comment:
Don't really like this, but I wanted to use a component since these are all going to be so similar. I can make a separate component for each one with different headers if you all think it'd be best.